### PR TITLE
I've added class for uppercase first character

### PR DESCRIPTION
**Pull request by bekliev**

### CODE REVIEW

**Comment by cvrebert** [comment]
This is approximately a v4 equivalent of #18279. Caveats similar to what Patrick mentioned there would apply here.

File: scss/_utilities.scss [hunk 1]
```diff
@@ -71,6 +71,7 @@
 .text-lowercase      { text-transform: lowercase !important; }
 .text-uppercase      { text-transform: uppercase !important; }
 .text-capitalize     { text-transform: capitalize !important; }
+.text-ucfirst        { &:first-letter { text-transform: uppercase; } }
```
Comment (cvrebert) [inline scss/_utilities.scss]: Use the psuedo-element form, i.e. `&::first-letter` with two colons

Comment (BeklievParviz) [inline scss/_utilities.scss]: Why?

Comment (cvrebert) [inline scss/_utilities.scss]: The pseudo-class form is deprecated and all of our supported browsers support the newer pseudo-element form.

Comment (BeklievParviz) [inline scss/_utilities.scss]: Where can I know more about it?

Comment (cvrebert) [inline scss/_utilities.scss]: https://developer.mozilla.org/en-US/docs/Web/CSS/::first-letter

**Comment by mdo** [comment]
Passing on this—I'm unsure how much use folks would really expect to get out of this as a utility.

**Pull request closed without merging**